flow/Arena.h

```diff
@@ -242,14 +242,18 @@ class Optional : public ComposedIdentifier<T, 4> {
 	}
 
 	bool present() const { return impl.has_value(); }
-	T& get() {
+	T& get() & {
```

Comment:
This is indeed a good learning about _ref-qualifier_, see _dcl.fct_, _over.load_ and _over.match.funcs_ in N4713.